doc/release.md

```diff
@@ -147,7 +158,6 @@ cp dist/sourmash*tar.gz ../../../testenv3/
 cd ../../../testenv3/
 deactivate
 source bin/activate
-python -m pip install pytest build wheel
```

Comment:
`pip install -r requirements.txt` below will install `pytest`, and we don't need the others